inc/customizer/custom-controls/color-group/color-group.scss

```
@@ -0,0 +1,27 @@
+.customize-control.customize-control-ast-color-group {
+    display: flex;
+    flex-wrap: wrap;
+    .ast-toggle-desc-wrap {
+        flex: 1 1 auto;
+    }
+    .ast-field-color-group-wrap {
+        justify-content: flex-end;
+        display: flex;
+    }
+    .color-button-wrap {
+        position: inherit;
+    }
+    .color-group-item:not(:last-child) {
+        margin-right: 8px;
+    }
+    .astra-popover-color {
+        position: absolute;
+        left: 0;
+        z-index: 9;
+    }
+    .ast-color-group-reset {
+        right: 82px;
+        top: 7px;
+        cursor: pointer;
+    }
+}
```

```suggestion
}

```

Can you confirm you have editor-config extension installed for your IDE and that is using the `.editorconfig` file?

---

It was not installed previously, just installed now & corrected this one.